<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Energy state space</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0">
    <widget class="QCustomPlot" name="mainData" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_8" columnstretch="2,1">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_7" columnstretch="8,1">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_6" rowstretch="5,1">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="beta_label">
               <property name="text">
                <string>β</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="mu_label">
               <property name="text">
                <string>μ</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QDoubleSpinBox" name="beta_spinbox">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>30.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>0.600000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="mu_spinbox">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-10.000000000000000</double>
               </property>
               <property name="maximum">
                <double>0.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>-5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QSlider" name="beta_slider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>30000</number>
               </property>
               <property name="value">
                <number>600</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="mu_slider">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
               <property name="value">
                <number>50000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>10000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="sample_label">
             <property name="text">
              <string>Sample name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sample_lineEdit">
             <property name="text">
              <string>aerogel_bcc_b_18_0.05</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QComboBox" name="runType_comboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>basic</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="y_lineEdit">
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="drawLimits_checkBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Limits</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="aerogelOrIsing_checkBox">
           <property name="text">
            <string>Aerogel</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="fps_label">
             <property name="text">
              <string>FPS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="fps_lcd">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="dataPackSize_label">
           <property name="text">
            <string>Data Pack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataPackSize_lineEdit">
           <property name="text">
            <string>100000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="maxCells_label">
           <property name="text">
            <string>Max cells</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxCells_lineEdit">
           <property name="text">
            <string>400000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QPushButton" name="print_button">
           <property name="text">
            <string>Print</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="stop_button">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="fullView_button">
           <property name="text">
            <string>Full view</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="focus_button">
           <property name="text">
            <string>Focus</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="start_button">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="clear_button">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="pps_label">
           <property name="text">
            <string>PPS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="pps_lcd">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="digitCount">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
